cmake/FindLeapMotion.cmake

```diff
@@ -117,6 +117,7 @@ endif()
 find_library(LEAPMOTION_LEAP_LIBRARY_RELEASE
     NAMES
     Leap
+    libLeap
```

Comment:
Does CMake not automatically/suitable prepend `lib` on platforms where that is standard? I thought they did, but I have a bunch of modules with lines like this too, so presumably there must be some cases where it doesn't work right.